src/main/scala/org/clulab/habitus/utils/DefaultContextExtractor.scala

```diff
@@ -106,4 +116,26 @@ class DefaultContextExtractor extends ContextExtractor {
     }
     mentionsWithContext.distinct
   }
+
+  def yearToString(yearOpt: Option[Int]): String = {
+    val year = yearOpt.getOrElse("N/A")
+    year.toString
+  }
+}
+
+object DefaultContextExtractor {
+
+  val config = ConfigFactory.load()
+  val localConfig: Config = config[Config]("VarReader")
+  val inputDir: String = localConfig[String]("regionsLexicon")
+  val files = FileUtils.findFiles(inputDir, ".tsv")
+  val regionMap = mutable.Map[String, String]()
+  files.foreach { f =>
+    val source = Source.fromFile(f)
```

Comment:
Here's where I wonder how this would work from a jar file.  Perhaps that will never happen.

Reply:
Should I file an issue for now to investigate later? To be honest, I am not sure how this should be addressed :(

Reply:
That sounds good.  If you eventually describe the situation a little more, about when these regions are being processed, it would help.  It looks like it's reading a directory, but resources don't work quite like that.  Are we expecting users to edit the configuration file that is contained inside the jar file in order to specify the directory?  Things like that.

Reply:
I think I was thinking that that place in the resources is where all regions file would go, like lexicons do, and users would not be changing that location. 
I was hoping the regions files would be read once in that default context object and then we would just use the created map. 

> but resources don't work quite like that
Would you please say more on this?